PatientWebApplication/HealthClinic.DAL/Service/OperationService.cs

```diff
@@ -55,7 +58,68 @@ public Boolean isTermNotAvailable(DoctorUser doctor, TimeSpan start, TimeSpan en
             if (hasAppointmentDoctor || hasOperationDoctor ) return true;
             return false;
         }
-       
+
+        public List<Operation> GetOperationsForPatient(int id)
+        {
+            return _operationRepository.GetOperationsForPatient(id);
+        }
+
+        public List<Operation> SimpleSearchOperations(AppointmentReportSearchDto appointmentReportSearchDto)
+        {
+            List<Operation> operations = GetOperationsForPatient(appointmentReportSearchDto.PatientId);
+
+            operations = SearchForDate(operations, appointmentReportSearchDto);
+
+            operations = SearchForDoctorNameAndSurname(operations, appointmentReportSearchDto);
+
+            operations = SearchForAppointmentType(operations, appointmentReportSearchDto);
```

Comment:
Write in the form:
"List<Operation> operations = SearchForAppointmentType(SearchForDoctorNameAndSurname(SearchForDate(GetOperationsForPatient(appointmentReportSearchDto.PatientId), appointmentReportSearchDto), appointmentReportSearchDto), appointmentReportSearchDto);"